From 8939853850637a2a42ad4df751991dc377eda0e5 Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 11 Jan 2018 08:20:24 +0200
Subject: [PATCH 09/19] legacy: label old hardcoded data paths

Commit fcc42312d7f8fe1a665d121d0d9f405b4c0c5fea removed
the old location data paths.

Commit 575d2497e377f95ee2d751b0cfe8fed014ca8dc2 removed
the ims_rtpd socket label.

Commit 121430c5598c37748b81bb74ad2c3ec426a67414 removed
the old port_bridge data path.

Commit fa220b8117fda15ab98f94b9350eb7be82ca0c2b removed
the old ipacm data path.

Commit b72606ae88a57248c1c16d3b807e9b4f962e63a9 removed
the old iop data path.

Change-Id: I86de41ac9721394dd0c805a80bacb02f8a7722d1
---
 legacy-common/file_contexts | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/legacy-common/file_contexts b/legacy-common/file_contexts
index b2703d0..8ac20c6 100644
--- a/legacy-common/file_contexts
+++ b/legacy-common/file_contexts
@@ -2,3 +2,20 @@
 /(vendor|system/vendor)/bin/perfd     u:object_r:perfd_exec:s0
 /data/system/perfd(/.*)?              u:object_r:mpctl_data_file:s0
 /dev/socket/perfd                     u:object_r:mpctl_socket:s0
+
+# Location
+/data/misc/location(/.*)?                                           u:object_r:location_data_file:s0
+/data/misc/location/mq/location-mq-s                                u:object_r:location_socket:s0
+/data/misc/location/mq/alarm_svc                                    u:object_r:location_socket:s0
+
+# IMS
+/dev/socket/ims_rtpd                            u:object_r:ims_socket:s0
+
+# Port bridge
+/data/misc/port_bridge(/.*)?                                        u:object_r:port_bridge_data_file:s0
+
+# IPA
+/data/misc/ipa(/.*)?                                                u:object_r:ipacm_data_file:s0
+
+# IOP
+/data/misc/iop(/.*)?                                                u:object_r:iop_data_file:s0
-- 
2.7.4

